autocorpus/section.py

```diff
@@ -7,13 +7,128 @@
 """
 
 import re
+from importlib import resources
+from typing import Any
 
 import nltk
 from fuzzywuzzy import fuzz
 
 from . import logger
 from .references import References
-from .utils import handle_not_tables, read_iao_term_to_id_file, read_mapping_file
+from .utils import handle_not_tables
+
+
+def read_mapping_file() -> dict[str, list[str]]:
+    """Reads the IAO mapping file and parses it into a dictionary.
+
+    Returns:
+        The parsed IAO mappings
+    """
+    mapping_dict: dict[str, list[str]] = {}
+    mapping_path = resources.files("autocorpus.IAO_dicts") / "IAO_FINAL_MAPPING.txt"
+    with mapping_path.open(encoding="utf-8") as f:
+        lines = f.readlines()
+        for line in lines:
+            heading = line.split("\t")[0].lower().strip("\n")
+            iao_term = line.split("\t")[1].lower().strip("\n")
+            if iao_term != "":
+                if "/" in iao_term:
```

Comment:
This bit could be simplified. Just split `iao_term` on `"/"` and add all the results to the dict